### DISCUSSION TITLE
DITA-OT <hazardstatement> Colon and SVG Sizing Issue

### DISCUSSION THREAD

**Discussion by ShayneDoan** [dita-ot/dita-ot#4550] [Q&A]
Hi all,

The first problem occurs when using the <hazardstatement> element in DITA and publishing as a PDF or as an XHTML output. The <hazardstatement> correctly does not have a colon in the header. However, when uploading to an external site like FluidTopics, the <hazardstatement> has a colon in the header that I would like to remove. 

The second problem consists of the <hazardstatement> image attached in the first column. The PDF and XHTML output correctly resizes the image but when uploading to an external site, the image is too big and makes the table uneven. Is there a way to see the CSS transformations from the DITA-OT applied to a <hazardstatement> image so I can copy the changes in the external site?

Attached is the XHTML output, PDF output, and FluidTopics output.

XHTML Output
![XHTMLOutput](https://github.com/user-attachments/assets/ca551aad-28f2-486b-8a17-2055a316981c)

PDF Output
![PDFOutput](https://github.com/user-attachments/assets/60d38080-2ea3-4026-8d35-87880d6d2a84)

FluidTopics Output
![Fluidtopics Output](https://github.com/user-attachments/assets/c94ef1f7-2729-423e-a7a6-f39156603137)

**Comment by raducoravu** [comment]
Maybe you can search in all of the DITA OT files for the string "topic/image" (without the quotes) and you should find all XSL templates which process image elements (for various publishing plugins). Also maybe if you contact FluidTopics directly they might be willing to assist further as the problem seems to be in their publishing plugins.
